frontend/lib/norent/tests/i18n-provider-for-tests.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { mergeIntoLinguiCatalog, li18n } from "../../i18n-lingui";
+import { I18nProvider } from "@lingui/react";
+import * as enCatalog from "../../../../locales/en/norent.chunk";
+
+export const NorentI18nProviderForTests: React.FC<{
```

Comment:
Bleh I don't really like this but it was the quickest way to make the translations available to some of our components so our snapshot tests would work.

Reply:
Seems good to me!